scripts/globals/battlefield.lua

```diff
@@ -57,6 +57,17 @@ xi.battlefield.leaveCode =
     LOST   = 4
 }
 
+xi.battlefield.dropChance =
+{
+    EXTREMELY_LOW  = 2,
+    VERY_LOW       = 10,
+    LOW            = 30,
+    NORMAL         = 50,
+    HIGH           = 70,
+    VERY_HIGH      = 100,
+    EXTREMELY_HIGH = 140,
+}
+
```

Comment:
Where is this used? I cant see it ><

Reply:
https://github.com/LandSandBoat/server/pull/2785/files#diff-a986f244ec3d08b169692e68724208e36214c7c5307dcf4b48cb4ee22bfaa4f1R270